src/Files.App/UserControls/SideBar/SideBarView.xaml

```diff
@@ -75,7 +75,7 @@
 						ElementPrepared="MenuItemsHost_ElementPrepared"
 						IsTabStop="True"
 						ItemTemplate="{StaticResource DefaultSidebarItemTemplate}"
-						ItemsSource="{x:Bind ViewModel.SidebarItems, Mode=OneWay}"
+						ItemsSource="{x:Bind MenuItemsSource, Mode=OneWay}"
```

Comment:
Is this needed for the fix?

Reply:
Yes, to access owner's menu list. Being inside of SidebarViewModel makes each item inaccessible to it.